tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/tb/AbstractTitleBlockTestCase.java

```diff
@@ -67,6 +68,9 @@ public void testDiagramTitleBlock() {
   }
 
   public void testElementTitleBlock(String elementId) {
+	// unsynchronized
+	DiagramHelper.setSynchronized(diagram.getDiagram(), false);
+
     // create TB tools
     DAnnotation elementTB = diagram.createElementTitleBlock(elementId);

```

Comment:
Need to test also the synchronized mode, you can move here the line to set the diagram in unsync mode, then create a new element TB (to cover both cases).